po/fr.po

```diff
@@ -0,0 +1,64 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the gamemode-extension package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+# Translators:
+# Christian Kellner <christian@kellner.me>, 2019
```

Comment:
You should probably take credit for your work here ;) and maybe we should fill out the `SOME DESCRIPTIVE TITLE` and following lines as well (I guess I missed that myself in the other file). Ups.